share/governance.md

```diff
@@ -0,0 +1,111 @@
+
+Vox Pupuli Governance
+====================
+
+
+Roles and Responsibilities
+--------------------------
+
+In order to have a smoothly running project, formal roles with corresponding responsibilities are established. A member of the community may have multiple roles.
+
+
+Users
+-----
+
+*How to become one*: Use a Vox Pupuli project
+
+
+Users are community members who have a need for the project. They are the most important members of the community: without them, the project would have no purpose. Anyone can be a user; there are no specific requirements. Users should be encouraged to participate in the life of the project and the community as much as possible. User contributions enable the project team to ensure that they are satisfying the needs of those users. Common user activities include (but are not limited to):
+
+* Advocating for use of the project
+* Informing developers of project strengths and weaknesses from a user’s perspective
+* Providing moral support (a ‘thank you’ goes a long way)
+* Writing documentation and tutorials
+* Filing bug reports and feature requests
+* Participating on the discussion board
+
+
+Users who continue to engage with the project and its community will often find themselves becoming more and more involved. Such users may then go on to become collaborators and contributors, as described below.
+
+
+
+Collaborators
+-------------
+
+*How to become one*: Submit a pull request to a Vox Pupuli project
+
+Collaborators are community members who submit patches to the project. These patches may be a one-time occurrence or occur over time. It is anticipated that collaborators' contributions may be small at first, growing more considerable once the collaborator has built up the necessary knowledge, experience, and confidence.
+
+
+
+Contributors
+------------
+
+*How to become one*: Be a collaborator and be nominated to the PMC as a contributor. Nominations should be sent to <email address for nominations>. You may nominate yourself. It is also common to give contributor status to an individual who donates code to the project by migrating a repository to the github namespace.
```

Comment:
I don't think we need to. People can find the definition of the PMC if they keep reading, or ^F search for it.